proto/v1beta1/discovery.proto

```diff
@@ -84,4 +84,7 @@ message Discovered {
   // details to show to the user. The LocalizedMessage is output only and
   // populated by the API.
   google.rpc.Status analysis_status_error = 4;
+
+  // Output only. The time occurrences related to this discovery occurrence were archived.
```

Comment:
Use field behavior OUTPUT_ONLY instead of the comment. https://google.aip.dev/203#output-only

See https://github.com/grafeas/grafeas/blob/master/proto/v1beta1/spdx.proto#L271